pkg/install/resources.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 the Heptio Ark contributors.
+Copyright 2018 the Velero contributors.
```

Comment:
I'm not on top of this discussion, should we be making this change now? If so, should the same change be made on the other files?

Reply:
good q, we should try to be more on top of this. Any time we touch a file, we should update it, so this one should technically be:
```
Copyright 2018, 2019 the Velero contributors.
```

Reply:
If it was simply `Heptio Ark -> Velero` I'd say we could do a single find/replace, but since we need to append rather than replace the year I'm not sure if it's worth modifying them in bulk. I'm guessing the regex wouldn't be too complicated, but also...regex.

Reply:
One pass: `s/Copyright 2018 the Heptio Ark/Copyright 2018, 2019 the Velero`, another pass: `s/Copyright 2017 the Heptio Ark/Copyright 2017, 2019 the Velero`. Should cover all?